samples/Playground/Playground.Windows/Playground.Windows.csproj

```diff
@@ -22,6 +22,7 @@
 
 	<ItemGroup>
 		<None Remove="appsettings.platform.json" />
+		<None Remove="ThemeSwitchPage.xaml" />
```

Comment:
Changes to this csproj file shouldn't be required

Reply:
moved files to shared proj